catalogue/webapp/pages/progress.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import PageWrapper from '@weco/common/views/components/PageWrapper/PageWrapper';
+import {spacing, grid} from '@weco/common/utils/classnames';
+
+const ProgressPage = () => (
+  <div className={`row ${spacing({s: 5}, {padding: ['top']})}`}>
```

Comment:
Could/should this Christmas tree of divs be replaced with one of our Layout components?

Reply:
Probably, the copy pasting, reformating nunjucks => jsx has just left me really bored of it.

Reply:
Testify!